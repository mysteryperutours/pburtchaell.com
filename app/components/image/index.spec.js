import React from 'react';
import Image from './index';

describe(Image.name, () => {
  it('should render component', () => {
    expect(shallow(
      <Image
        src="http://path/to/image"
        alt="Foo"
        height="0"
        width="0"
      />
    )).toMatchSnapshot();
  });
});
