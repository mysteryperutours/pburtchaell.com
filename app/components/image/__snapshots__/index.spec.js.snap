exports[`Image should render component 1`] = `
<div
  className="image-preload-container is-pending"
  style={
    Object {
      "height": "0px",
      "width": "0px",
    }
  } />
`;
